stdlib/public/core/Arrays.swift.gyb

```diff
@@ -1138,21 +1147,21 @@ extension ${Self} : RangeReplaceableCollection, _ArrayProtocol {
   /// The total number of elements that the array can contain using its current
   /// storage.
   ///
-  /// If the array grows larger than its capacity, it discards its current
-  /// storage and allocates a larger one.
+  /// If you add more elements to an array than it can hold with its current
+  /// capacity, it discards its current storage and allocates a larger one.
```

Comment:
I borrowed a paragraph from the type discussion that better communicates this bit.